### DISCUSSION TITLE
Remapping of some groups in Frigate install

### DISCUSSION THREAD

**Discussion by remz1337** [tteck/Proxmox#2886] [General]
Why do we need to remap those groups? In my experience, the container works fine with the default mapping.

https://github.com/tteck/Proxmox/blob/46be5a9b744cfc8691dab056f819572be84139c1/install/frigate-install.sh#L97

**Comment by tteck** [comment]
It will work fine if you're not utilizing hardware acceleration. The LXC `render` group has to match the Host `render` group in order to allow hardware acceleration. https://github.com/tteck/Proxmox/issues/2523

**Reply by remz1337** [reply]
Indeed, maybe we can make this statement conditional to HW acceleration being available. A few way to check this here: https://wiki.debian.org/HardwareVideoAcceleration

```
apt-get install -y mesa-utils
HWA = $(glxinfo | grep "direct rendering: Yes" | cut -d ":" -f 2 | sed "s| ||g")
if [ HWA == "Yes" ]; then
    # Enable HW accel
    # Remap GID
fi
apt-get remove -y mesa-utils
```

**Reply by tteck** [reply]
Remapping those groups doesn't bother anything if you're not utilizing hardware acceleration.
